CMakeLists.txt

```
@@ -1,11 +1,15 @@
 cmake_minimum_required( VERSION 3.5 FATAL_ERROR )
```

I know this isn't quite the point of this PR, but since we are in here, we should probably bump this to 3.10.


```suggestion
cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
```

---

done in https://github.com/ros/urdfdom/pull/236/commits/3c14334856ab2742bf30706f70018411517e13d3